backend/tests/init_data/skills/sandbox/test_provider.py

```
@@ -0,0 +1,27 @@
+
+def test_prepare_base_params_includes_skill_identity_token():
+    """Sandbox provider should forward skill identity token from context."""
+    provider = SandboxToolProvider()
+    context = SkillToolContext(
+        task_id=1,
+        subtask_id=2,
+        user_id=3,
+        db_session=None,
+        ws_emitter=None,
+        user_name="alice",
+        auth_token="task-jwt",
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to argument: "auth_token"

(S106)